src/main/resources/scripts/vanilla gui.sk

```diff
@@ -0,0 +1,15 @@
+#
+# A vanilla example of a gui creation using Skript only
+# Requiert Skript 2.3+
```

Comment:
How would the user have this script without skript 2.3 :P

Reply:
> How would the user have this script without skript 2.3 :P

With only Skript? He can't.

The user can't make it run without skript 2.3+, but if this pull request is merged, it should not be a problem 'cause it will be Skript 2.4+

Reply:
Exactly my point, this will come bundled with skript 2.4, no need to add a skript 2.3 notice

Reply:
Yeah but I prefer to add this precision if any user want to share the example.

Reply:
In that case, it's in french not english :p

Reply:
omg I didn't see this fault, sorry about that, I will fix it.